nemo/collections/nlp/modules/common/megatron/layer_norm_1p.py

```diff
@@ -23,6 +24,9 @@
 
     HAVE_APEX = True
 except (ImportError, ModuleNotFoundError):
+    import torch.nn.functional as F
+    from torch.nn.modules.normalization import LayerNorm
+
```

Comment:
What about "OrigFastLayerNorm" symbol in except path?

Reply:
I don't follow the question, we don't use "OrigFastLayerNorm" here in the except path without apex